.coveragerc

```diff
@@ -4,4 +4,10 @@ exclude_lines =
     if __name__ == .__main__.:
 
 # Only show one number after decimal point in report.
-precision = 1
\ No newline at end of file
+precision = 1
+
+[run]
+omit =
+    models/

```

Comment:
We have tests covering models, why are we omitting them from coverage reports?

Reply:
they are not unit tests